js_src/pid_display.js

```diff
@@ -17,42 +17,45 @@
  * Copyright (C) 2011 Simon Newton
  */
 
+goog.provide('app.MessageStructure');
+
 goog.require('goog.dom');
 goog.require('goog.ui.Component');
 goog.require('goog.ui.Tooltip');
 
 
-goog.provide('app.pid');
-
-
 /**
  * A message field, this represents a field within a RDM message.
+ * @param {*} field_info
+ * @param {?goog.dom.DomHelper=} opt_domHelper Optional DOM helper.
  * @constructor
  * @extends goog.ui.Component
  */
-app.pid.MessageField = function(field_info, opt_domHelper) {
```

Comment:
I think this should be able to switch back to app.pid.MessageField now the circle has been broken.

Reply:
But it was just `app.MessageField` before my changes?

Reply:
Ah sorry, I must have been looking at an intermediate diff before.